src/validate.rs

```diff
@@ -16,9 +16,7 @@ pub fn validate_credential_path(credential_path: &str) -> Result<(), ValidationE
         || credential_path.starts_with('/')
         || credential_path.starts_with('?'))
     {
-        Err(ValidationError::CredentialPathNotAllowed(
-            credential_path.to_string(),
-        ))
+        Err(ValidationError::CredentialPathNotAllowed)
```

Comment:
This is more relevant to https://github.com/internet-computer-protocol/ic-eth-rpc/pull/147, but these PRs are all kind of interdependent since they all rely on [!17319](https://gitlab.com/dfinity-lab/public/ic/-/merge_requests/17319).